src/main/java/rx/Scheduler.java

```diff
@@ -43,6 +43,17 @@
  *    maintenance.
  */
 
+    /** 
+     * The tolerance for a clock drift in nanoseconds where the periodic scheduler will rebase. 
+     * <p>
+     * The associated system parameter, {@code rx.scheduler.drift-tolerance}, expects its value in minutes.
+     */
+    static final long CLOCK_DRIFT_TOLERANCE_NANOS;
+    static {
+        CLOCK_DRIFT_TOLERANCE_NANOS = TimeUnit.MINUTES.toNanos(
```

Comment:
yep, ^.

BTW, @akarnokd don't you think that minutes is not the best unit of time measurement in programming? I'd prefer millis, `long` seems enough for `CLOCK_DRIFT_TOLERANCE`, and anyway, you're converting it to nanos then so `long` is 100% enough for millis.

Also, `15` minutes seems too much for a default clock drift tolerance…